internal/tools/common/common.go

```diff
@@ -0,0 +1,52 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Instead of creating an overly-generic and unspecific `common` package here, can we just put this code in `internal/tools/repo.go`?

Reply:
I was hoping to add a few more common functions (such as funcs for finding all modules file paths and paths in the repo, etc). However, I guess they would all be relating to use for just within the repo. I'm not sure, so do you think that renaming the package to `repo` would be better?

Reply:
Why not just pull these tools in the `tools` package like mentioned? I don't see a need for an additional package here.

Reply:
Good question; it seems like currently [the `tools` package has a build tag](https://github.com/open-telemetry/opentelemetry-go/blob/a113856a76005325689fb5364851af677778d9a3/internal/tools/tools.go#L15), which prevents it from being called from the scripts within `internal/tools/`. I believe that this build tag is used for managing the internal/external tooling, so @Aneurysm9 suggested putting the common funcs within a different package.

Reply:
If you put the code a file other than the `tools.go` file, i.e. `repo.go`, in the `internal/tools` package and do not include that build tag in that new file there should be no issue using the code.

Reply:
Oh, awesome! Didn't know that 👍 
If I understand correctly, I would be move the `common.go` file within the `internal/tools` directory, and then change the declaration to `package tools`, and it would be imported as `"go.opentelemetry.io/otel/internal/tools` (without the `common` on the end). I'll work on that later today. Thanks!